components/src/components/SplitButton/SplitButton.tsx

```diff
@@ -21,12 +21,29 @@ const MainButton = styled(Button)`
     z-index: 0;
   }
 `;
-const OptionButton = styled(Button)`
+
+type OptionButtonProps = {
+  purpose: ButtonPurpose;
+};
+
+const OptionButton = styled(Button)<OptionButtonProps>`
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
+
+  ${props =>
+    props.purpose !== 'secondary' &&
+    `
+    border-left: 1px solid white;
```

Comment:
istedenfor `white` lag `SplitButton.tokens.tsx` fil og ha med en design token fra `ddsBaseTokens` for border. Eventuelt hent token for tekstfarge fra `Button`, siden det er derfor border er hvit. Og så bruk den i styling her. Vi skal ha ulike themes etter hvert og vi trenger en struktur på alt som bruker tokens.

```suggestion

// SplitButton.tokens.tsx
import { ddsBaseTokens } from '@norges-domstoler/dds-design-tokens';

const mainButton = {
  primary: {
    border-left: `1px solid ${ddsBaseTokens.colors.DdsColorNeutralsWhite}`
  }
} 
    export const tokens = {
    primaryButton
    }
    
  // SplitButton.tsx  
  border-left: ${tokens.mainButton.primary.borderLeft};
```